nameko/testing/services.py

```diff
@@ -261,43 +294,45 @@ def cm_to_inches(self, cms):
                 return self.maths_rpc.divide(cms, 2.54)
 
         container = ServiceContainer(ConversionService, config)
-        maths_rpc = replace_dependencies(container, "maths_rpc")
+        mock_maths_rpc = replace_dependencies(container, "maths_rpc")
 
         container.start()
 
         with ServiceRpcProxy('conversionservice', config) as proxy:
             proxy.cm_to_inches(100)
 
         # assert that the dependency was called as expected
-        maths_rpc.divide.assert_called_once_with(100, 2.54)
+        mock_maths_rpc.divide.assert_called_once_with(100, 2.54)
 
-    """
-    if container.started:
-        raise RuntimeError('You must replace dependencies before the '
-                           'container is started.')
+    **Kwargs Usage**
```

Comment:
I think we can keep the original heading as **Usage** and just give this an introductory sentence:

> Providing a specific replacement by keyword: